src/processor/tasks.py

```diff
@@ -148,15 +147,12 @@ def check_deleted_wikipedia_references():
     db.session.commit()
 
 
-@celery_app.task(name='send-metrics-to-metrics-api')
-def send_metrics_to_metrics_api(send_limit=None):
-    """ Send metrics to the metrics API using nameko."""
+@celery_app.task(name='send-metrics')
+def send_metrics():
+    """Send all metrics collected over the past day to the metrics-api. """
 
-    metrics_service = ServiceHandler(service=MetricsAPIServiceDispatcher)
-    for event in Event.query.all()[:send_limit]:  # temp send limit
-        data = prepare_metrics_data(
-            uri=event.uri.raw,
-            origin=event.origin,
-            created_at=event.created_at
-        )
-        metrics_service.send(data)
+    events_to_send = Event.query.filter(
+        Event.last_updated >= utcnow().shift(days=-1).datetime
```

Comment:
This should be fine, as long as the service receiving the message on the other side is able to deal with the same event being sent twice.

Reply:
Basically, it will fail for that Event because of the duplication; but that is fine since it does not affect other (new) events going through. 
Once the 'official' `metrics-api` is released we can modify it to fail more gracefully if needed.